plugins/extra/cannonjs/componentConfigs/CannonBodyConfig.ts

```diff
@@ -11,8 +16,15 @@ export default class CannonBodyConfig extends SupCore.Data.Base.ComponentConfig
         z: { type: "number", mutable: true },
       }
     },
-
-    shape: { type: "enum", items: ["box", "sphere", "cylinder"], mutable: true },
+    orientation: {
```

Comment:
Maybe orientationOffset would be clearer ? Maybe we should rename offset to positionOffset as well.